reaktive/src/commonTest/kotlin/com/badoo/reaktive/maybe/DoOnAfterCompleteTest.kt

```diff
@@ -17,7 +17,7 @@ class DoOnAfterCompleteTest : MaybeToMaybeTests by MaybeToMaybeTestsImpl({ doOnA
 
     @Test
     fun does_not_call_action_WHEN_upstream_succeeded() {
-        val isCalled = AtomicBoolean()
+        val isCalled = AtomicBoolean(false)
```

Comment:
It's `false` by default. Why is this change?

Reply:
I see IDE warning/error (I'm on the EAP). Let me revert such changes.

Reply:
Done. Here is error:

<img width="829" alt="Screenshot 2021-10-31 at 17 17 56" src="https://user-images.githubusercontent.com/295424/139588022-1a978b2a-5631-4555-9c1d-4e17c6f3907b.png">